wzdx/standard_to_enhanced/planned_events_translator.py

```diff
@@ -0,0 +1,393 @@
+
+import argparse
+import json
+import logging
+import copy
+from collections import OrderedDict
+from os import path
+import uuid
+
+from wzdx.sample_files.validation_schema import wzdx_v40_feed, road_restriction_v40_feed
+
+from wzdx.tools import date_tools, polygon_tools, wzdx_translator
+
+PROGRAM_NAME = 'PlannedEventsTranslator'
+PROGRAM_VERSION = '1.0'
+
+
+DEFAULT_PLANNED_EVENTS_FEED_INFO_ID = '49253be7-0c6a-4a65-8113-450f9041f989'
+
+
+def main():
+    input_file, output_file = parse_planned_events_arguments()
+    # Added encoding argument because of weird character at start of incidents.xml file
+
+    planned_events_obj = json.loads(open(input_file, 'r').read())
+    wzdx, event_type = wzdx_creator(planned_events_obj)
```

Comment:
The `wzdx_creator` method seems to return a Dict object, but here we're trying to set two variables?